src/test/java/tests/navigation/GoToShopWithDniproCity.java

```diff
@@ -0,0 +1,26 @@
+package tests;
+
+import base.AbstractBaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.AlloUaHomePageElements;
+
+public class GoToShopWithDniproCity extends AbstractBaseTest {
+@Test
+    public void GoToStores(){
+        AlloUaHomePageElements allo = new AlloUaHomePageElements(driver);
+        driver.navigate();
+        GoToAllo();
+        sleep(3);
```

Comment:
Are you need sleep(3)? Try without sleep. Are you need driver.navigate? Explain what is it.